synapse/lib/stormtypes.py

```diff
@@ -5269,6 +5282,43 @@ async def _methNodeGlobTags(self, glob):
                     ret.append(groups)
         return ret
 
+    @stormfunc(readonly=True)
+    async def _methNodeDiffTags(self, names, prefix=None, apply=False):
+        newtags = set()
+        names = set(await toprim(names))
+        tagpart = self.valu.snap.core.model.type('syn:tag:part')
+
+        async for name in toiter(names):
```

Comment:
Yea, i suspect we can't do this because the expectation will be that you can feed the results of `tags()` to this API or even use it to diff tags at different nesting levels. Lets have a quick goback :)

Reply:
Yeah the intent there was to make it behave similar to `$lib.tags.prefix` and prevent additional levels from being created. I was considering making the tagpart norming an optional behavior, which i think would make it suitable for both the 'unknown tags from an API' and 'safe list of tags' purposes?

Reply:
Updated to remove norming